### DISCUSSION TITLE
Draggable connections + canvas configuration

### DISCUSSION THREAD

**Discussion by alex-d-ivan** [Foblex/f-flow#78] [Feature Request] [answered]
1st feature: I couldn't find this feature but it would be nice if the connections could be dragged. Right now if we move a group of nodes the connections start to be messy, so a means to drag them around and position them nicely would be great.

2nd feature that I think would be nice is to configure the sizes of the canvas so the sizes of nodes can be resized automatically based on the size of the canvas. Example: If we need to print a node tree on a paper format, if we can configure the canvas size and then the nodes are resized accordingly the tree would be easy to print.

**Comment by siarheihuzarevich** [comment, marked as answer]
Hi. Cool suggestions, maybe someday it will be implemented. Thanks for the idea
